www.foia.gov/_sass/base/_typography.scss

```diff
@@ -0,0 +1,4 @@
+$font-weight-1: 900;
+$font-weight-2: 700;
+$font-weight-3: 600;
+$font-weight-4: 500;
```

Comment:
I just realized there is also a `_typography.scss` file in `theme`. Does it make sense to combine them?

Reply:
✔️